templates/common/storybook/.storybook/main.js

```diff
@@ -1,8 +1,4 @@
 module.exports = {
-  stories: [
```

Comment:
How we will identify the story book files?

Reply:
const req = require.context("../../", true, /story\.tsx$/);

on Preview.js I have added context to be root... which includes app and packages

Reply:
Cool